apps/frontend/src/welcome/__tests__/Welcome.test.tsx

```diff
@@ -14,11 +14,6 @@ beforeEach(() => {
   nock(API_BASE_URL).get('/users/invites/42').reply(200, {});
 });
 
-// storage
```

Comment:
Pretty sure this was obsolete, at least it was the only place we used `sessionStorage` lol